components/molecules/SortedBySelector/sorted-by-selector.tsx

```diff
@@ -0,0 +1,50 @@
+import ContextFilterButton from "components/atoms/ContextFilterButton/context-filter-button";
+
+import React, { useEffect, useRef, useState } from "react";
+import Icon from "../../atoms/Icon/icon";
+import cancelIcon from "public/x-circle.svg";
+import Radio from "components/atoms/Radio/radio";
+import humanizeNumber from "lib/utils/humanizeNumber";
+import getFilterKey from "lib/utils/get-filter-key";
+
+interface SortedBySelectorProps {
+  handleCancelClick: () => void;
+  className?: string;
+  selected?: string;
+}
+
+const SortedOrders = {
+  "name": "Repository",
+  "prsCount": "PR Count",
+  "prVelocityCount": "PR Velocity",
+  "spamPrsCount": "Spam"
+};
+
+const SortedBySelector: React.FC<SortedBySelectorProps> = ({
+  handleCancelClick,
+  selected
+}) => {
+
+  if (!selected) return null;
```

Comment:
This must return an element, even if its empty

```suggestion
  if (!selected) return <></>;
```